src/Components/Publishing/ToolTip/LinkWithTooltip.tsx

```diff
@@ -44,10 +47,11 @@ export class LinkWithTooltip extends Component<Props, State> {
   }
 
   componentDidMount() {
-    if (this.link) {
-      const position = findDOMNode(this.link).getBoundingClientRect()
-      this.setState({ position })
-    }
+    const SetupToolTipPosition = () => defer(this.setupToolTipPosition)
```

Comment:
Aliases to other methods are typically prefixed with an underscore so that most of the body still matches 1:1 -- `_setupToolTipPosition`